### PR TITLE
Remove arangojs reference documentation

### PR DESCRIPTION
**Pull request by Simran-B**
Link to external docs: http://arangodb.github.io/arangojs/

### CODE REVIEW

**Comment by pluma** [comment]
Please do not merge this before 7.0.0 final is released. The documentation is currently under development and final URLs may change.

File: 3.7/aql/common-errors.md [hunk 1]
```diff
@@ -64,8 +64,8 @@ Assembling query strings with simple **string concatenation** looks trivial,
 but is potentially **unsafe**. You should use
 [bind parameters](fundamentals-bind-parameters.html) instead whenever possible,
 use query building functionality if provided by a driver (see
-[arangojs AQL Helpers](../drivers/js-reference-aql.html) for example)
-or at least sanitize user input with great care.
+[arangojs AQL Helpers](http://arangodb.github.io/arangojs/modules/_aql_.aql.html){:target="_blank"}
```
Comment (Simran-B) [inline 3.7/aql/common-errors.md:67]: Shall we point to the devel version or find another solution that allows us to link to the latest stable version?
```suggestion
[arangojs AQL Helpers](http://arangodb.github.io/arangojs/devel/modules/_aql_.aql.html){:target="_blank"}
```

File: 3.7/drivers/js-getting-started.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ description: ArangoJS is compatible with the latest stable version of ArangoDB a
 ArangoJS is compatible with the latest stable version of ArangoDB available at
 the time of the driver release.
 
-The [_arangoVersion_ option](js-reference-database.html)
+The [_arangoVersion_ option](http://arangodb.github.io/arangojs/modules/_connection_.html#config){:target="_blank"}
```
Comment (Simran-B) [inline 3.7/drivers/js-getting-started.md:12]:
```suggestion
The [_arangoVersion_ option](http://arangodb.github.io/arangojs/devel/modules/_connection_.html#config){:target="_blank"}
```

File: 3.7/drivers/js-getting-started.md [hunk 3]
```diff
@@ -187,8 +187,10 @@ const db = new Database({
 });
 ```
 
-For AQL please check out the [aql template tag](js-reference-database-queries.html#aql) for writing parametrized
-AQL queries without making your code vulnerable to injection attacks.
+For AQL please check out the
+[aql template tag](http://arangodb.github.io/arangojs/modules/_aql_.aql.html){:target="_blank"}
```
Comment (Simran-B) [inline 3.7/drivers/js-getting-started.md:191]:
```suggestion
[aql template tag](http://arangodb.github.io/arangojs/devel/modules/_aql_.aql.html){:target="_blank"}
```

**Review by Simran-B — COMMENTED** [review]
Changes need to be applied to 3.8 once they are ready

**Pull request merged**